Blog post for Spring 176 (#476)
openSUSE/obs-landing: _posts/development/2024-10-23-make-it-easier-for-users-to-understand-what-is-there-on-a-request.md

---
layout: post
title: 3 Improvements For Collaboration on the Request Page
category: development
---

> Our recent redesign of the request page has introduced three significant improvements that make it easier for you to understand better their requests. Those improvements are part of the [beta program](/2018/10/04/the-beta-program/).

Reviewer: ```suggestion
Today we want to tell you about three significant improvements we have just introduced that make it easier to collaborate on the request page. Those improvements are part of the [beta program](/2018/10/04/the-beta-program/).
```

{% include partials/_series-of-posts-about-request-redesign.md %}

> # More Clarity on the Diffs

Reviewer: ```suggestion
## More Clarity on the Diffs
```

The diff tab now shows small highlights (called eyecatchers) that show the exact change. Those eyecatchers will help you see what exactly is new or changed and allow you to focus better on those differences, making it easier to review the request.

<figure>
  <img src="/images/posts/2024-10-23/eyecatchers.png" alt="Eyecatchers in a file diff" />
  <figcaption>Eyecatchers in a file diff</figcaption>
</figure>

> # Streamlined Conversations

Reviewer: ```suggestion
## Less Clutter in Conversations
```

> You'll notice that the review decision and the comment boxes are now combined into just one box. This will reduce the clutter on a request's conversation timeline and help you focus on _what_ you have to do, not _where_.

Reviewer: Not sure I understand what you want to say with this.

Author: Before we had two boxes right? If you wanted to write a comment, you had  to find the comment box. If you wanted to give a reason because you declined something you had to find the "decision box". Now, you only have one box, so you don't have to think where you write things anymore because there is no choice. Makes sense?

<figure>
  <img src="/images/posts/2024-10-23/decision_and_comment_boxes_combined.png" alt="Review decisions and comment boxes combined" />
  <figcaption>Review decisions and comment boxes combined</figcaption>
</figure>

> # Less Noise on the Builds

Reviewer: ```suggestion
## Less Noise in the Build Results
```

> The build status of a request was showing a lot of information that was not really useful for you, like the "Excluded" state. Now it shows only relevant information to the changes. This will make easier for you to understand the state of the request and help you make a decision about it.

Reviewer: ```suggestion
The build results of a request were showing a lot of information that is not really useful to decide if you should merge the request or not, like the "Excluded" state. We limited the build results in the overview to succeeded/failed and building now. This will make easier for you to understand the state of the request and help you make a decision about it.
```

---
layout: post
title: Make it easier for users to understand what is there on a request
category: development
---
Our recent redesign of the request page has introduced three significant improvements that make it easier for users to understand better their requests.

> This blog post is part of the [beta program](/2018/10/04/the-beta-program/).

Reviewer: A blog post is not part of the beta program...

{% include partials/_series-of-posts-about-request-redesign.md %}

# More Clarity on the Diffs

> We've added eyecatchers on the request's diff page. The diff component now shows small highlights that help users understand what exactly changed before and after.

Reviewer: ```suggestion
We've added eyecatchers on the request's diff page. The diff component now shows small highlights that help users understand what exactly changed.
```
